ydb/core/audit/audit_log_impl.cpp

```diff
@@ -112,19 +110,34 @@ TString GetJsonLogCompatibleLog(const TEvAuditLog::TEvWriteAuditLog::TPtr& ev) {
     return ss.Str();
 }
 
-TString GetTxtLog(const TEvAuditLog::TEvWriteAuditLog::TPtr& ev) {
-    const auto* msg = ev->Get();
+TString GetTxtLog(TInstant time, const TAuditLogParts& parts) {
     TStringStream ss;
-    ss << msg->Time << ": ";
-    for (auto it = msg->Parts.begin(); it != msg->Parts.end(); it++) {
-        if (it != msg->Parts.begin())
+    ss << time << ": ";
+    for (auto it = parts.begin(); it != parts.end(); it++) {
+        if (it != parts.begin())
             ss << ", ";
         ss << it->first << "=" << it->second;
     }
     ss << Endl;
     return ss.Str();
 }
 
+// Array of functions for converting TEvAuditLog::TEvWriteAuditLog events to a string.
```

Comment:
Not a `TEvAuditLog::TEvWriteAuditLog events` anymore but `audit log record` -- `converting audit log record to a string`.